pubspec.yaml

```
@@ -20,7 +20,7 @@ dev_dependencies:
   flutter_lints: ^5.0.0
 
 # The following section is specific to Flutter packages.
-flutter: {}
+flutter: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)